### PR TITLE
Revisions to FMC 102 (Authentication tokens)

### PR DESCRIPTION
**Pull request by klevenstein**
* General edits
* Expanding the Postman procedures with more detail
* Restructing of lab into 3 parts
* Retitling to remove "102"

### CODE REVIEW

**Comment by klevenstein** [comment]
There's still one section that's incomplete; am getting more info to contextualize the Python script in 3.md.

File: labs/firepower-restapi-102/2.md [hunk 1]
```diff
@@ -1,134 +1,21 @@
-# Refreshing an Authentication Token for FMC REST API
-Firepower Management Center REST API authentication tokens are valid for 30 minutes, and can be refreshed up to three times.
+# Requesting a FMC REST API authentication token with Postman
 
-Obtain valid authentication and refresh tokens[explained on previous page] from the Firepower Management Center REST API. Ensure these tokens have been refreshed less than three times.
+Before you begin, ensure that REST API is enabled in your FMC. This procedure is documented in [REST API Fundamentals Learning Lab](https://learninglabs.cisco.com/tracks/devnet-beginner/rest-api-fundamentals/what-are-rest-apis/step/1a).
```
Comment (annegentle) [inline labs/firepower-restapi-102/2.md:4]: I think it's documented in the prior lab for FMC itself.

File: labs/firepower-restapi-102/3.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+4. In the **Save Request** dialog, give your request a name (such as "FMC Token Refresh") and create or select a collection or folder where the request will be saved.
+5. At the top of the request tab, select **POST** from the drop-down menu and enter the following URL: `https://<management_center_IP_or_name>/api/fmc_platform/v1/auth/refreshtoken`.
+6. Select the **Headers** tab. Under **Key**, enter `x-auth-access-token` and under **Value** enter the corresponding token value. Do the same for `x-auth-refresh-token`.
+8. Click **Send**.
+
+The response will appear under **Headers**. The response includes a new access token and a new refresh token.
+
+## Authentication to the FMC REST API
+
+The following Python script defines
```
Comment (annegentle) [inline labs/firepower-restapi-102/3.md]: missing rest of sentence.

File: labs/firepower-restapi-102/firepower-restapi-102.json [hunk 3]
```diff
@@ -30,11 +30,14 @@
         "title": "Cisco Security REST API"
     }],
     "files": [{
-        "title": "FMC REST API 102",
+        "title": "# Firepower Management Center (FMC) REST API authentication tokens",
```
Comment (annegentle) [inline labs/firepower-restapi-102/firepower-restapi-102.json]: You can remove the # mark.

**Review by annegentle — CHANGES_REQUESTED** [review]
This is a nice re-write! Minor comments sprinkled within.

**Pull request merged**